DPG/Tasks/AOTTrack/tagAndProbeDmesons.cxx

```diff
@@ -560,6 +561,20 @@ struct ProbeThirdTrack {
     trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDca].SetMaxChi2PerClusterITS(36.f);
     trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDca].SetMaxDcaZ(2.f);
 
+    // TPC tracks (global tracks without ITS IB requirement)
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetTrackType(o2::aod::track::TrackTypeEnum::Track);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetPtRange(0.1f, 1e10f);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetEtaRange(-0.8f, 0.8f);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetRequireITSRefit(true);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetRequireTPCRefit(true);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetRequireGoldenChi2(true);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetMinNCrossedRowsTPC(70);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetMinNCrossedRowsOverFindableClustersTPC(0.8f);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetMaxChi2PerClusterTPC(4.f);
+    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetRequireHitsInITSLayers(3, {0, 1, 2, 3, 4, 5, 6}); // at least three hits in whatever layer
```

Comment:
```suggestion
    trackSelector[aod::tagandprobe::TrackTypes::GlobalWoDcaWoItsIb].SetRequireHitsInITSLayers(4, {0, 1, 2, 3, 4, 5, 6}); // at least three hits in whatever layer
```

Take into account this suggestion if and only if the goal is to study the matching efficiency between a TPC track and an ITS track. Leaving `3` in principle there might be also "afterburnt" tracks

Reply:
ok you replied me privately, approving